components/profileshaders.tsx

```diff
@@ -1,3 +1,4 @@
+import { Button } from '@mui/material';
```

Comment:
Please import Button directly like the other imports. When you do it like this it slows down builds quite a lot (which is stupid but we have to deal with it): https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
Fixed.